css/main.css

```diff
@@ -39,16 +40,33 @@ header {
   line-height: 24px;
 }
 
-.menu-bars {
-  background: transparent;
-  border: none;
-  outline: none;
+.container {
+  display: inline-block;
+  cursor: pointer;
 }
 
-.menu-bars i {
-  width: 18px;
-  height: 12px;
-  color: white;
+.bar1,
+.bar2,
+.bar3 {
```

Comment:
Good work, stacking your classes to reuse your attributes. 😎
and you used that many times across your CSS. 💯 

 _Awesome_